src/dsp_tools/models/sipi.py

```diff
@@ -27,7 +27,11 @@ def upload_bitstream(self, filepath: str) -> dict[Any, Any]:
             files = {
                 "file": (os.path.basename(filepath), bitstream_file),
             }
-            response = requests.post(self.sipi_server + "/upload?token=" + self.token, files=files)
+            response = requests.post(
+                self.sipi_server + "/upload?token=" + self.token,
+                files=files,
+                timeout=5 * 60,
```

Comment:
do you think that 5 minutes are a good timeout for uploading big multimedia files?

Reply:
hard to say... ideally, this wouldn't be hard coded but configurable. But for now, 5 minutes can't be all bad and it can always be changed if needed.